src/components/Testing/ControlsTest.tsx

```diff
@@ -40,28 +40,43 @@ export const ControlsTest = forwardRef<ControlsTestRef, ControlsTestProps>(
       setData: updateParams,
     }))
 
+    const allParamsCodeDriven = definition?.every(
+      (d) => d.update === "code-driven"
+    )
+
     return (
       <div className={classes.container}>
+        {allParamsCodeDriven && (
+          <p className={classes.codeDrivenNote}>
+            <i className="fa-solid fa-triangle-exclamation" aria-hidden />
+            <span>
+              All params of this artwork are defined as "code-driven". This will
+              enable a dedicated minting experience for collectors.
+            </span>
+          </p>
```

Comment:
When testing params and all params are code driven we notify the user that we detected the dedicated minting experience